tests/dags/providers/provider_api_scripts/test_finnish_museums.py

```diff
@@ -282,34 +291,26 @@ def test_get_object_list_return_none_if_none_json():
 
 
 def test_process_object_with_real_example():
```

Comment:
Do you mind adding a test for `get_creator` which tests the output if a creator is supplied for each type?

Reply:
Added in [`5740895`](https://github.com/WordPress/openverse-catalog/pull/978/commits/5740895b60d15397fca29d53afa75f90705df92e)!